packages/css/src/components/avatar/avatar.scss

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+.ams-avatar {
+  aspect-ratio: 1/1;
+  background-position: center;
```

Comment:
Can we separate the rules you need for an Avatar with an image from the rules for an Avatar without?

Reply:
I think this is no longer relevant.